shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/helper/extension/String+Regex.kt

```diff
@@ -0,0 +1,8 @@
+package co.nimblehq.avishek.phong.kmmic.helper.extension
+
+fun String.isValidEmail(): Boolean {
+
+    val expression = "\\b[\\w\\.-]+@[\\w\\.-]+\\.\\w{2,4}\\b"
+    val regex = Regex(expression)
+    return isNotEmpty() && matches(regex)
```

Comment:
We can use the `toRegex()` method here.
```suggestion
    return isNotEmpty() && matches(expression.toRegex())
```